apps/token/src/routes/governance/components/proposal-detail-header/proposal-header.spec.tsx

```diff
@@ -187,29 +203,26 @@ describe('Proposal header', () => {
     );
     // For a rationale over 100 chars, we expect the header to be truncated at
     // 100 chars with ellipsis and the details-one element to contain the rest.
-    expect(screen.getByTestId('proposal-header')).toHaveTextContent(
-      'Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Aenean…'
-    );
+    expect(screen.getByTestId('proposal-header')).toHaveTextContent('0x0');
     expect(screen.getByTestId('proposal-details-one')).toHaveTextContent(
-      'dolor'
+      'Class aptent taciti sociosqu ad litora torquent per conubia' // shorten to 60 characters
```

Comment:
Chore (non-blocking): I think you can remove this comment, the string shortening is happening in proposal-header